src/traces/pie/attributes.js

```diff
@@ -79,7 +79,27 @@ module.exports = {
 
     text: {
         valType: 'data_array',
```

Comment:
Oh sorry, that's `label`... I still don't see a good usage for a single-value `text` but perhaps it would be better to define it consistently with other trace types anyway.